docs/frontend/guides/how-we-use-react-query.md

```diff
@@ -143,16 +143,60 @@ const onSubmit = async () => {
 :::
 
  Callbacks maybe not fire as expected. To avoid that issue, logic should be handled in the `useMutation` callback which is called first. 
- ```jsx reference
-https://github.com/transcom/mymove/blob/main/src/pages/Office/EditShipmentDetails/EditShipmentDetails.jsx#L22-L30
+
+```js 
+ import {useQueryClient} from '@tanstack/react-query'
+
+const queryClient = useQueryClient();
+
+const { mutate: myNewMutation } = useMutation(
+   {
+  mutationFn: functionToBeCalled ,
+    // First callback
+   onSuccess: () => { 
+       console.log('Where logic should be handled') 
+       queryClient.invalidateQueries({queryKey: [myKey]})
+       },
+   onError: () => {
+       console.log('Error handling');
+       }
+   });
```

Comment:
Can we make these references too? I'm concerned with the code copied here being out of date going forward.

Reply:
The references were moved to the section below with added context.